config/default/common/config/wv.json/stories/default/worldview_layerpicker.json

```diff
@@ -0,0 +1,189 @@
+              {
+                "target": "#events-sidebar-tab",
+                "title": "Events",
+                "content": "Select the Events tab to view a curated list of current natural events around the world.",
+                "placementBeacon": "right"
+              },
+              {
+                "target": "#download-sidebar-tab",
+                "title": "Data Download",
+                "content": "Select the Data tab to download the underlying data for the imagery layers displayed on the map. Once you have selected your desired layer it will direct you to Earthdata Search to download the data.",
+                "placementBeacon": "right"
+              },
+              {
+                "target": "#timeline",
+                "title": "Timeline",
+                "content": "The Timeline allows you to choose the date of observation and much more!",
+                "placementBeacon": "right"
+              },
+              {
+                "target": "#date-selector-main",
+                "title": "Date Selector",
+                "content": "Change the date (and time if you have Geostationary layers loaded) by selecting and typing into the date fields, clicking on the up and down arrows by each field.",
+                "placementBeacon": "right"
+              },
+              {
+              "target": "#timeline-interval-btn-container",
+              "title": "Interval Selector",
+              "content": "Select the timeline interval you are interested in, or set your own.",
+              "placementBeacon": "right"
+            },
+            {
+              "target": "#zoom-buttons-group",
+              "content": "Use the back and forward arrows to step backward or forward in time according to the selected interval.",
+              "placementBeacon": "right"
+            },
+              {
+                "target": "#animate-button",
+                "title": "Timeline Animation",
+                "content": "Animate the imagery by clicking on the video camera icon. Click on Play in the animation box to watch the imagery change day by day. You can even  export it as an animated GIF.",
+                "placementBeacon": "right"
+              },
+              {
+                "target": "#timeline-layer-coverage-panel-handle",
+                "title": "Timeline Layer Coverage",
+                "content": "Click here to open the timeline layer coverage panel. Temporal availability of active time-varying layers are shown as a set of striped horizontal blue lines in the panel. The total temporal coverage of the layers is shown on the far right.",
+                "placementBeacon": "right"
+              },
+              {
+                "target": ".zoom-level-change",
+                "title": "Timescale Zoom",
+                "content": "Adjust the timeline view to view the timescale in years, months, and days (And, hours and minutes if you have a Geostationary layer loaded).",
```

Comment:
Calling this a "zoom" feels a little off to me.  Maybe we can just refer to it as the "timescale" since that's what the label in the component calls it?  So perhaps "Adjust the timescale to years, months or days..." etc

Reply:
I agree "timescale" sounds better. One less "zoom" the better

Reply:
used Jason's suggestion